src/Raven.Server/Documents/Subscriptions/Stats/SubscriptionConnectionRunStats.cs

```diff
@@ -0,0 +1,22 @@
+// -----------------------------------------------------------------------
+//  <copyright file="SubscriptionConnectionRunStats.cs" company="Hibernating Rhinos LTD">
+//      Copyright (c) Hibernating Rhinos LTD. All rights reserved.
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+
+namespace Raven.Server.Documents.Subscriptions.Stats
+{
+    public class SubscriptionConnectionRunStats
+    {
+        public long TaskId { get; set; }
+        public string TaskName { get; set; }
+        
+        public string ClientUri { get; set; }
+        public DateTime ConnectedAt { get; set; }
+
+        public string Exception { get; set; }
+        public long BatchCount { get; set; }
```

Comment:
I wonder if we should include total batch sizes as well (w/o documents counters ts drill down). It might be useful for gaps to find out if batches are heavy of not.

Reply:
done - added